app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java

```diff
@@ -154,10 +164,17 @@
     private String reminder;
     private int count = 0;
     private static final String[] id = new String[1];
+
+    private SpeechRecognizer recognizer;
+    /* Keyword we are looking for to activate menu */
+    private static final String KEYPHRASE = "hi susi";
+    /* Named searches allow to quickly reconfigure the decoder */
+    private static final String KWS_SEARCH = "hi susi";
```

Comment:
@chiragw15 @jyoth1raditya @betterclever After testing the app, I found that it only recognized 2 to 3 times out of ten. The reason I found is the change in accents. On testing with other speech to text engines when we say "Hi Susi" It actually interprets it as "Hi Sushi". Due to which the feature does not work at most times. The probable solution could be to have a list of Strings of probable phrases similar to "Hi Susi" and then match it.